validator/src/bin/solana-test-validator.rs

```diff
@@ -247,6 +247,17 @@ fn main() {
                      If the ledger already exists then this parameter is silently ignored",
                 ),
         )
+        .arg(
+            Arg::with_name("clone_account_from_file")
+                .long("clone-from-file")
+                .value_name("PATH")
+                .takes_value(true)
+                .multiple(true)
```

Comment:
Let's ensure the caller provides a valid account address:
```suggestion
                .validator(is_pubkey_or_keypair)
                .multiple(true)
```